### PR TITLE
Support allowScripts for Angular 14

### PR DESCRIPTION
**Pull request by OlegKipchatov**

### CODE REVIEW

File: packages/devextreme-schematics/src/utility/patch.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+
+export class CustomNodePackageInstallTask extends NodePackageInstallTask {
+    toConfiguration = (): any => {
+        const rootOptions = super.toConfiguration();
+        const resultObject = {
+            name: rootOptions.name,
+            dependencies: rootOptions.dependencies,
+            options: Object.assign({}, rootOptions.options, { allowScripts: true })
```
Comment (dxvladislavvolkov) [inline packages/devextreme-schematics/src/utility/patch.ts:9]: may be smth like: { ...rootOptions.options,  allowScripts: true } ?

File: packages/devextreme-schematics/src/utility/patch.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+
+export class CustomNodePackageInstallTask extends NodePackageInstallTask {
+    toConfiguration = (): any => {
+        const rootOptions = super.toConfiguration();
+        const resultObject = {
+            name: rootOptions.name,
+            dependencies: rootOptions.dependencies,
```
Comment (dxvladislavvolkov) [inline packages/devextreme-schematics/src/utility/patch.ts:8]: Do rootOptions have dependencies property? I don't see it here https://github.com/angular/angular-cli/blob/main/packages/angular_devkit/schematics/tasks/package-manager/install-task.ts#L56

**Review by dxvladislavvolkov — CHANGES_REQUESTED** [review]

File: packages/devextreme-schematics/src/install/index.ts [hunk 3]
```diff
@@ -12,22 +12,20 @@ import {
...
 
-import {
-  NodePackageInstallTask
-} from '@angular-devkit/schematics/tasks';
-
 import { latestVersions } from '../utility/latest-versions';
 import { modifyJSONFile } from '../utility/modify-json-file';
 import { getProjectName } from '../utility/project';
 
+import * as patch from '../utility/patch';
```
Comment (dxvladislavvolkov) [inline packages/devextreme-schematics/src/install/index.ts:19]: You can use named import: import { CustomNodePackageInstallTask } from '../utility/patch'

**Review by dxvladislavvolkov — CHANGES_REQUESTED** [review]

File: packages/devextreme-schematics/src/install/index.ts (hunk 3 above)
Comment (OlegKipchatov) [reply packages/devextreme-schematics/src/install/index.ts]: https://github.com/DevExpress/devextreme-cli/pull/793/commits/0527ba123b3e8516dfcfe824a318010c487fb3a5

**Review by dxvladislavvolkov — APPROVED** [review]

**Pull request merged**